examples/wistia/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "wistia",
+  "version": "1.0.0",
+  "description": "Vistia  Widget for contentful",
+  "main": "./src/app.js",
+  "scripts": {
+    "env": "env",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "widget:create": "./node_modules/.bin/better-npm-run widget:create",
+    "widget:update": "./node_modules/.bin/better-npm-run widget:update",
+    "widget:dev": "./node_modules/.bin/better-npm-run widget:dev",
+    "widget:serve": "./node_modules/.bin/better-npm-run widget:serve"
+  },
+  "betterScripts": {
+    "widget:create": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget create --space-id $SPACE_ID"
+    },
+    "widget:update": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget update --space-id $SPACE_ID --force"
+    },
+    "widget:dev": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget update --space-id $SPACE_ID --force --src \"http://localhost:3030/index.html\""
```

Comment:
> npm better scripts seem to run in a "subshell". Hence I need to re-export the credentials as otherwise widget cli fails...maybe I am overlooking something though...?

Try this
~~~
"betterScripts": {
  "env": {
    "command": "env | grep ACCESS_TOKEN"
  }
}
~~~
If you set the access token in `.env` it will show up as an environment variable. This is enough for `contentful-widget`. So we can safely remove the re-export of the token.